src/code/views/weather-station-view.tsx

```diff
@@ -96,18 +102,35 @@ export class WeatherStationView extends
         fontSize: "36pt",
         fontWeight: "bold",
         alignSelf: "flex-end"
+      },
+      button: {
+        gridRow: "5",
+        gridColumn: "1",
+        boxShadow: "none",
+        position: "relative",
+        width: 160
       }
     };
 
     const settings = simulation && simulation.settings,
-          showTemperature = settings && settings.showTempValues;
-
-    function renderTemperature() {
-      return showTemperature
-              ? `${unitTempStr || ''}`
-              : null;
-    }
-    const precip = (weatherStation && weatherStation.precipitation)  ? "Rain" : "Clear";
+          showTemperature = settings && settings.showTempValues,
+          hasPresence = !!(simulation && simulation.selectedPresence),
+          temperatureStr = hasPresence
+                            ? (showTemperature ? `${unitTempStr || ''}` : null)
+                            : "Good \u00A0bye!",
+          precip = weatherStation
+                    ? (weatherStation.precipitation  ? "Rain" : "Clear")
+                    : "",
+          exitButton = hasPresence
+                        ? <RaisedButton
+                            className={"weather-station-exit-button"}
+                            style={styles.button}
+                            disabled={!hasPresence}
+                            onClick={this.handleExit}
+                            label="\u274C\u00A0\u00A0Sign Out"
```

Comment:
What are these characters?

Reply:
I added some named constants to clarify the intent. In doing so, I realized that if you hover over a string constant in Visual Studio Code that contains unicode escapes, a tooltip appears which shows the resulting Unicode string.